dm/config/subtask.go

```diff
@@ -53,14 +53,19 @@ type RawDBConfig struct {
 }
 
 // DefaultRawDBConfig returns a default raw database config
-func DefaultRawDBConfig(readTimeout string) *RawDBConfig {
+func DefaultRawDBConfig() *RawDBConfig {
 	return &RawDBConfig{
-		ReadTimeout:  readTimeout,
 		MaxIdleConns: defaultMaxIdleConns,
 	}
 }
 
-// SetWriteTimeout adds writeTimeout for raw database
+// SetReadTimeout set readTimeout for raw database
```

Comment:
for `raw database config`, right?

ditto for `SetWriteTimeout`.

Reply:
sure, fix [it](https://github.com/pingcap/dm/pull/266/files#diff-94b1e02e1fe1bd32d9eee7a40a730b00R62-R74)